plugins/org.jboss.tools.jst.web.ui/src/org/jboss/tools/jst/web/ui/WebUiPlugin.java

```diff
@@ -264,6 +264,14 @@ public static JSPTextEditor getActiveEditor() {
 		return editor;
 	}
 
+	public static JSPMultiPageEditor getActiveMultiPageEditor() {

```

Comment:
This method is used by https://github.com/jbosstools/jbosstools-vpe/pull/297/files#diff-5
We do need it to get access to our multipageeditor here. If there is no our multipageeditor we will get null which is fine.